README.md

```diff
@@ -27,111 +29,148 @@ conda env create -f environment.yml
 conda activate scitt
 ```
 
-## Clone the emulator
+## Clone the Emulator
 
-Clone the scitt-api-emulator repository and change into the scitt-api-emulator folder:
+1. Clone the scitt-api-emulator repository and change into the scitt-api-emulator folder:
 
-```sh
-git clone https://github.com/scitt-community/scitt-api-emulator.git
-```
-or for ssh:
+    ```sh
+    git clone https://github.com/scitt-community/scitt-api-emulator.git
+    ```
 
-```sh
-git clone git@github.com:scitt-community/scitt-api-emulator.git
-```
-then:
+1. Move into the emulator director to utilize the local commands
 
-```sh
-cd scitt-api-emulator
-```
+    ```sh
+    cd scitt-api-emulator
+    ```
 
-## Start the proxy server
+## Start the Proxy Server
 
 The proxy server supports 2 options currently:
- - 'CCF' uses the emulator server to create and verify receipts using the CCF tree algorithm
- - 'RKVST' uses the RKVST production SaaS server to create and verify  receipts using native Merkle trees
 
-Note the emulated is for experimentation only and not recommended for production use.
+- 'CCF' uses the emulator server to create and verify receipts using the CCF tree algorithm
+- 'RKVST' uses the RKVST production SaaS server to create and verify  receipts using native Merkle trees
 
-### Start a fake emulated SCITT service
+**Note:** _the emulator is for experimentation only and not recommended for production use._
 
-```sh
-./scitt-emulator.sh server --workspace workspace/ --tree-alg CCF
-```
+### Start a Fake Emulated SCITT Service
 
-### Start an RKVST SCITT proxy service
+1. Start the service, under the `/workspace` directory, using CCF
 
-```sh
-./scitt-emulator.sh server --workspace workspace/ --tree-alg RKVST
-```
+    ```sh
+    ./scitt-emulator.sh server --workspace workspace/ --tree-alg CCF
+    ```
+
+1. The server is running at http://localhost:8000/ and uses the `/workspace` folder to store the service parameters and service state  
```

Comment:
Minor-nit: although I understand what you mean, writing `/workspace` with the leading slash implies an absolute path from the root of whatever filesystem the emulator code is cloned to, did you mean `workspace/` given the above command?

```suggestion
1. The server is running at http://localhost:8000/ and uses the `workspace/` folder to store the service parameters and service state  
```